### PR TITLE
Updated feedback in postprocessing image

### PR DESCRIPTION
**Pull request by staalezh**

### CODE REVIEW

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 1]
```diff
@@ -2,52 +2,126 @@
...
+        ground_truth=ground_truth,
+        dataset_id=dataset_id
+    )
+
+
+def post_feedback_v2(las_client: las.Client, document_id: str, dataset_id: str, feedback: dict):
+    def should_post_feedback(item, label):
+        return not item[label].get('automated') or feedback[label].get('isEdited')
+
+    ground_truth = []
```
Comment (augustak) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:37]: Ground truth for line items need to have a nested structure similar to predictions V1 format

**Review by augustak — CHANGES_REQUESTED** [review]

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 2]
```diff
@@ -2,52 +2,126 @@
...
+
+    for label in feedback:
+        is_line_item = isinstance(feedback[label], list)
```
Comment (magnurud) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:40]: Use .items()  
```suggestion
    for label field_feedback in feedback.items():
        is_line_item = isinstance(field_feedback, list)
```

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 3]
```diff
@@ -2,52 +2,126 @@
...
+        if is_line_item:
+            if not feedback[label]:
+                # Ignore blank lines
+                continue
+            
+            lines = []
+            for line in feedback[label]:
+                line_gt = []
+                for line_label in line:
```
Comment (magnurud) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:49]: use .items()

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 4]
```diff
@@ -2,52 +2,126 @@
...
+                    if should_post_feedback(line, line_label):
```
Comment (magnurud) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:50]: line_label will never be a key in feedback

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 5]
```diff
@@ -2,52 +2,126 @@
...
+    las_client.update_document(
+        document_id=document_id,
+        ground_truth=ground_truth,
+        dataset_id=dataset_id
+    )
+
+
+def post_feedback_v2(las_client: las.Client, document_id: str, dataset_id: str, feedback: dict):
+    def should_post_feedback(item, label):
+        return not item[label].get('automated') or feedback[label].get('isEdited')
```
Comment (magnurud) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:35]:
```suggestion
    def should_post_feedback(item):
        return not item.get('automated', True) or item.get('isEdited', False)
```

**Review by magnurud — CHANGES_REQUESTED** [review]

File: docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py [hunk 6]
```diff
@@ -2,52 +2,140 @@
...
+    elif uri := os.environ.get('WEBHOOK_URI'):
+        webhook_endpoints = [{'uri': uri}]
 
-    response = {'documentId': document_id, 'datasetId': dataset_id, 'values': verified}
+    request_exception = None
+    for endpoint in webhook_endpoints:
+        logging.info(f'Posting result to {endpoint}...')
+        try:
+            requests.post(endpoint['uri'], json=response)
+        except requests.exceptions.RequestException as re:
```
Comment (augustak) [inline docs/workflows/transitions/postprocess/postprocess/feedback_and_export.py:135]:
```suggestion
        except (requests.exceptions.RequestException, KeyError) as re:
```

**Review by augustak — CHANGES_REQUESTED** [review]

**Review by magnurud — APPROVED** [review]

**Pull request merged**